src/lib/useDragAndZoom.tsx

```diff
@@ -36,6 +37,22 @@ export function fitBoundsInLimit(bounds: Bounds, limit: Bounds | undefined): Bou
     return bounds;
 }
 
+export function fitBoundsInMinVisibleX (newBounds: Bounds, oldBounds: Bounds, minVisibleX: number = 1): Bounds {
```

Comment:
You don't need default value `= 1` here since you never call this function without 3rd argument :)